Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js

```diff
@@ -301,8 +300,18 @@ localizedStrings["Cancellation @ Local Override Type"] = "Cancellation";
 localizedStrings["Canvas"] = "Canvas";
 localizedStrings["Canvas %d"] = "Canvas %d";
 localizedStrings["Canvas %s"] = "Canvas %s";
-/* Bitmap Renderer is a type of rendering context associated with a <canvas> element */
+/* 2D is a type of rendering context associated with a <canvas> element. */
+localizedStrings["Canvas Context Type 2D"] = "2D";
```

Comment:
We aren't super consistent about this, but should this be `2D @ Canvas Context Type` (same below)?